### PR TITLE
Features/remove migrations from scratch

### PR DESCRIPTION
**Pull request by ryanjorgeac**

### CODE REVIEW

File: .idea/dbnavigator.xml [hunk 1]
```diff
@@ -0,0 +1,489 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DBNavigator.Project.DataEditorManager">
+    <record-view-column-sorting-type value="BY_INDEX" />
+    <value-preview-text-wrapping value="false" />
+    <value-preview-pinned value="false" />
+  </component>
+<<<<<<< HEAD
```
Comment (petrolifero) [inline .idea/dbnavigator.xml:8]: This is rebase garbage, remove it.

File: Web/server.py [hunk 2]
```diff
@@ -1,25 +1,20 @@
...
-            dataAux = dataAux + data
-
-    print()
-    clientSocket.sendall(dataAux)
-    print(data.decode("utf-8"))
+    connection = httpConnection.httpConnection(clientSocket)
+    connection.process(bankApplication.root)
 
     clientSocket.close()
     serverSocket.close()
```
Comment (petrolifero) [inline Web/server.py:20]: if happens some exception on connection.process, this lines don't execute and a tcp port will not be freed, so use this sockets over a "with" statement

File: Web/test_HttpRequest.py [hunk 3]
```diff
@@ -11,37 +16,82 @@ def recv(self, bufsize, flags=0):
 
 
 def test_http_request_1():
+
     noTagsHeader = fakeSocket(b'GET /bank/Main-page HTTP/1.1\r\n\r\n')
-    request = httpRequest.httpRequest(noTagsHeader)
-    assert len(request.head().keys()) == 0
+    request = httpRequest.getNextHttpRequest(noTagsHeader, logging)
```
Comment (petrolifero) [inline Web/test_HttpRequest.py:21]: remove logging from here

File: Web/test_HttpRequest.py [hunk 4]
```diff
@@ -11,37 +16,82 @@ def recv(self, bufsize, flags=0):
...
+def test_http_request_getBody2():
+    oneTagHeader = {b'Content-length': b'20'}
+    socket = fakeSocket(b'a'*50)
+    body = httpRequest.getBody(socket, oneTagHeader, logging)
+    assert body == b'a'*20
+
+def test_http_request_getBody3():
+    oneTagHeader = {b'Content-length': b'20'}
+    socket = fakeSocket(b'a'*15)
+    try:
```
Comment (petrolifero) [inline Web/test_HttpRequest.py:81]: pytest know how to assert exceptions : https://docs.pytest.org/en/stable/reference/reference.html#pytest.raises
Use it.

File: Web/test_httpResponse.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    return httpResponse.httpResponse(headers, body, status)
+
+def test_httpResponse_1(responseExample):
+    assert responseExample.getStatus() == "200"
+
+def test_httpResponse_get_body(responseExample):
+    assert responseExample.getBody() == "<?xml version='1.0' encoding='UTF-8'?>"
+
+def test_httpResponse_responseAsBytes_Body(responseExample):
+    responseAsBytes = httpResponse.responseAsBytes(responseExample).split(b"\r\n\r\n")
```
Comment (petrolifero) [inline Web/test_httpResponse.py:19]: is Better not to split and compare the full return of method

File: condition.py [hunk 6]
```diff
@@ -1,4 +1,4 @@
-class condition():
```
Comment (petrolifero) [inline condition.py:1]: this file is really necessary?

**Review by petrolifero — CHANGES_REQUESTED** [review]
After resolve the threads, please remove garbage (binaries, log files and unused old files) from repository and them, in another commit, add them to gitignore to prevent them from enter. After that, make your IDE configuration don't enter in the repository too (https://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste)

File: Web/server.py (hunk 2 above)
Comment (ryanjorgeac) [reply Web/server.py:20]: Can I use a "with" statement inside another "with" statement?

File: .idea/dbnavigator.xml (hunk 1 above)
Comment (petrolifero) [reply .idea/dbnavigator.xml:8]: this <<<<<<< HEAD
and >>>>>>>
========34534534534

This kind of thing is not correct xml and was introduced by git rebase as conflicts that you don't resolve

File: Web/server.py (hunk 2 above)
Comment (petrolifero) [reply Web/server.py:20]: Yes

**Review by petrolifero — APPROVED** [review]

**Review by petrolifero — DISMISSED** [review]

**Review by petrolifero — APPROVED** [review]

File: .idea/.gitignore [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+<<<<<<< HEAD
```
Comment (petrolifero) [inline .idea/.gitignore:4]: remove this rebase garbage

File: .idea/.gitignore [hunk 8]
```diff
@@ -0,0 +1,7 @@
+# Default ignored files
```
Comment (petrolifero) [inline .idea/.gitignore:1]: this file was not removed by gitignore, so you must remove .idea/** gitignore entry, remove all files from .idea folder from index, and them add them again to gitignore, to remove them from repository

File: .gitignore [hunk 9]
```diff
@@ -0,0 +1,2 @@
+**/*.pyc
+.idea/**
```
Comment (petrolifero) [inline .gitignore:2]: the pyc ignore is consistent with the project, but idea is your IDE, so this kind of ignore (ignoring .idea folder) must not exist here, but in your personal gitignore

File: ApplicationService/transferFundsBetweenAccounts.py [hunk 10]
```diff
@@ -12,6 +12,11 @@ def execute(self, internalAccount, destinyLogin, amount):
         internalAccount.update(self.internalRepository)
         extAccount.update(self.externalRepository)
 
+    def execute(self, internalAccount, destinyLogin, amount):
```
Comment (petrolifero) [inline ApplicationService/transferFundsBetweenAccounts.py:15]: this method is duplicate, probably because a rebase incorrect. Resolve that

**Review by petrolifero — DISMISSED** [review]

**Pull request merged**